sktime/classification/tests/test_base.py

```diff
@@ -281,3 +283,21 @@ def _create_unequal_length_nested_dataframe(cases=5, dimensions=1, length=10):
         testy["dimension_" + str(i + 1)] = instance_list
 
     return testy
+
+
+MTYPES = ["numpy3D", "pd-multiindex", "df-list", "numpy3D"]
```

Comment:
numpy3D listed twice

Reply:
yes, that was a typo, should have been `"numpyflat"`.